src/components/Profile/ProfileBox.css

```diff
@@ -14,20 +14,28 @@
 
 .ProfileBox__Header {
   display: flex;
+  align-items: center;
+  justify-content: space-between;
 }
 
-.ProfileBox__Header > span {
+.ProfileBox__Header .ProfileBox__Header--title > span {
   font-weight: 500;
   font-size: 13px;
   line-height: 18px;
   text-transform: uppercase;
   color: var(--black-600);
-  margin-bottom: 13px;
   margin-right: 4px;
 }
 
+.ProfileBox__Header > div {
+  display: inline-flex;
+  text-transform: uppercase;
+}
+
+.ProfileBox__Container .Divider__Container {
+  margin: 13px 0;
+}
+
 .ProfileBox__Divider .Divider {
-  margin-bottom: 24px;
-  margin-top: 0;
   border-top: 1px solid var(--black-300);
```

Comment:
This class isn't necessary if you pass the color to the Divider as a prop.

Reply:
yeap, thanks. Merge error 😅